docs/content/how-to/cluster-configuration.md

```diff
@@ -0,0 +1,104 @@
+# Cluster Configuration
+
+## Overview
+
+In standalone OpenShift, cluster configuration is achieved via cluster-scoped resources in the `config.openshift.io/v1`
+package. Resources such as APIServer, OAuth, Proxy allow adding additional named certificates to the Kube APIServer, 
+adding identity providers, configuring the global proxy, etc. In Hypershift, configuration resources that
+impact the control plane need to be specified in the HostedCluster resource instead of inside the guest cluster. The
+resources still exist inside the guest cluster, but their source of truth is the HostedCluster and are continuously
+reconciled with what is specified in the HostedCluster.
+
+## Configuration in a HostedCluster
+
+The configuration resources that should be specified in the HostedCluster are:
+
+* APIServer - Provides API server configuration such as certificates and certificate authorities.
```

Comment:
might link each of these to the API ref in OCP docs? e.g. https://docs.openshift.com/container-platform/latest/rest_api/config_apis/apiserver-config-openshift-io-v1.html